SharperUniverse/Persistence/LiteDbProvider.cs

```diff
@@ -0,0 +1,119 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LiteDB;
+using SharperUniverse.Core;
+
+namespace SharperUniverse.Persistence
+{
+	public class LiteDbProvider : IDatabaseProvider
+	{
+		public string ConnectionString { get; set; }
```

Comment:
This won't ever change - the setter need not exist - make it so the fluent builder assigns this via the constructor instead.